source/server/lsp-message_loggers.adb

```diff
@@ -198,6 +203,69 @@ package body LSP.Message_Loggers is
       return Ada.Containers.Count_Type'Image (Value.Length);
    end Image;
 
+   -----------
+   -- Image --
+   -----------
+
+   function Image
+     (Value : LSP.Messages.ALS_Check_Syntax_Params) return String
+   is
+      use VSS.Strings;
+      use VSS.Strings.Conversions;
+      use VSS.String_Vectors;
+
+      function Reduce_Rules return String;
+      --  Joins the string elements seperated by a comma
+
+      ------------------
+      -- Reduce_Rules --
+      ------------------
+
+      function Reduce_Rules return String is
+         I : constant Reversible_Iterator := Value.Rules.Iterate;
+         L : constant Cursor := Last (I);
+         C : Cursor := First (I);
+         R : VSS.Strings.Virtual_String := VSS.Strings.Empty_Virtual_String;
+
+      begin
+         if Has_Element (C) then
+            while C /= L loop
+               Append (R, Value.Rules.Element (C));
+               Append (R, ", ");
+               C := Next (I, C);
+            end loop;
+
+            if Has_Element (C) then
+               Append (R, Value.Rules.Element (C));
+            end if;
+         end if;
+
+         return To_UTF_8_String (R);
+      end Reduce_Rules;
+
+   begin
+      return "Input: "
+        & To_UTF_8_String (Value.Input)
+        & "; Rules : ["
+        & Reduce_Rules
+        & "]";
+        --  & Value.Rules.Length'Image;
```

Comment:
commented out code, should be removed - or add a comment (for instance "uncomment this to debug")